lib/services.js

```diff
@@ -117,15 +123,43 @@ async function startService (udid, serviceName, socket, handshakeOnly = false) {
     } else {
       return await connectPort(udid, service.Port, socket);
     }
+  } catch (e) {
+    if (!tryMountIfFailed || serviceName === MOBILE_IMAGE_MOUNTER_SERVICE_NAME) {
+      throw e;
+    }
+    if (e.message?.includes('InvalidService') && autoMountDeveloperImage(udid)) {
+      return startService(udid, serviceName, socket, handshakeOnly, false);
+    }
   } finally {
     lockdown.close();
   }
 }
 
+async function autoMountDeveloperImage(udid) {
```

Comment:
This function should be documented

Reply:
also in readme

Reply:
Does it make sense if I move `autoMountDeveloperImage` to utilities and export as a function?

Reply:
yes, but then it won't be automount. We probably should rename it then to better describe the purpose